app/views/comments/_edit.html.erb

```diff
@@ -39,10 +39,6 @@
       </div>
     </div>
       <script type="application/javascript">
-        jQuery(document).ready(function() {
-          $E.initialize();
-        });
-
         function setInit(id) {
```

Comment:
I removed this `$E.initialize` call, I think it's unnecessary because it's already done in so many other places on a page with comments.